.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: ${{ needs.Get-CI-Image-Tag.outputs.ci-image-start-command }}
 
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

Comment:
We should mention why the downgrade is needed in the PR description for future reference

Reply:
Will update the description.